webapp/layouts/default.vue

```diff
@@ -4,30 +4,30 @@
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
           <ds-flex class="main-navigation-flex">
-            <ds-flex-item :width="{ lg: '3.5%' }" />
-            <ds-flex-item :width="{ base: '80%', sm: '80%', md: '80%', lg: '15%' }">
+            <ds-flex-item :width="{ base: '142px' }">
               <nuxt-link :to="{ name: 'index' }" v-scroll-to="'.main-navigation'">
                 <ds-logo />
               </nuxt-link>
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '20%', sm: '20%', md: '20%', lg: '0%' }"
+              :width="{ base: '40%', sm: '40%', md: '40%', lg: '0%' }"
               class="mobile-hamburger-menu"
             >
-              <ds-button icon="bars" @click="toggleMobileMenuView" right />
+              <base-button icon="bars" @click="toggleMobileMenuView" circle />
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '85%', sm: '85%', md: '50%', lg: '50%' }"
+              :width="{ base: '45%', sm: '45%', md: '45%', lg: '50%' }"
               :class="{ 'hide-mobile-menu': !toggleMobileMenu }"
+              style="flex-shrink: 0; flex-grow: 1;"
```

Comment:
why did you use inline styling in these cases? something to do with the styleguide?
is this to avoid targeting the class and applying `!important`... just guessing...

Reply:
It's to override inline styles applied by the styleguide – and I thought it's harder to miss this way than with applying a class and `!important` styles. Needs to be refactored either way!